Show cross linking containers links

rubenvp8510 opened the pull request:
![screenshot from 2017-02-23 00-44-26](https://cloud.githubusercontent.com/assets/3799716/23316645/e460bfa0-fa90-11e6-818c-e068386000f3.png)

mzazrivec commented on app/helpers/middleware_server_helper/textual_summary.rb:
```diff
@@ -62,16 +62,25 @@ def textual_version
-    lives_on_entity_name = _("Virtual Machine")
+
+    if lives_on.kind_of?(Container)
+      lives_on_entity_name = _("Container")
```
We should be having the whole label here, not just the entity name. I.e.:
```ruby
label = _("Underlying Container")
```

rubenvp8510 replied:
I agree, I'll do the change.

rubenvp8510 commented:
This PR depends on : https://github.com/ManageIQ/manageiq/pull/14043

bronaghs commented:
@miq-bot add_label middleware

mtho11 commented:
@miq-bot add_label ui, enhancement

abonas commented on app/helpers/middleware_server_helper/textual_summary.rb:
```diff
@@ -62,16 +62,25 @@ def textual_version
+    end
+
     {
-      :label => "Underlying #{lives_on_entity_name}",
```
Why is "underlying" not reused as previously? now it is repeated in line 70 and in line 73

mzazrivec replied:
Explained in a conversation above.

abonas commented:
@rubenvp8510 @mzazrivec  Is the review for this patch completed? it still says "changes requested", but it looks like all comments were addressed

abonas commented:
@rubenvp8510 @mzazrivec Is the review for this patch completed? it still says "changes requested", but it looks like all comments were addressed

rubenvp8510 commented:
@abonas from my point of view this is completed and ready for merge, I've already addressed the comments of @mzazrivec.

@mzazrivec  Is there more comments for this PR?

abonas commented:
@mzazrivec @martinpovolny @bronaghs if someone can review/ack this please

abonas commented:
@miq-bot add_label fine/yes
@simaishi @chrispy1, it was not labeled for fine although it should be in fine - it is part of a "must to have" feature  (we need to have some process ensuring things are labeled properly and not forgotten, especially since different teams are reviewing, acking and merging)

simaishi commented:
@rubenvp8510 @abonas we need a BZ to get this into `Fine`.  Please create one if it doesn't exist.

abonas commented:
@rubenvp8510 please see @simaishi comment. I was not sure if there's a BZ or only a JMAN? If no BZ, please create one

abonas commented:
@simaishi I added this BZ for backporting https://bugzilla.redhat.com/show_bug.cgi?id=1448070

chrispy1 commented:
@miq-bot add_label blocker

simaishi commented:
Fine backport details:

```
$ git log -1
commit 7236ecf8a6d0dfdeed426a9923cd558fd5927f3e
Author: Milan Zázrivec <mzazrivec@redhat.com>
Date:   Wed Apr 5 15:25:20 2017 +0200

    Merge pull request #484 from rubenvp8510/xlinking
    
    Show cross linking containers links
    (cherry picked from commit f2837ad74ccd7973877da507097c8a38c54f7acd)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1448131
```
